### PR TITLE
Worked on blocks

### PR DESCRIPTION
**Pull request by RandomAltThing**

### CODE REVIEW

**Comment by RandomAltThing** [comment]
The build is failing because the project itself is failing. This pull request is fine.

**Comment by FDKPIBC** [comment]
testplugins project lost some main files,so the build is failing

File: src/MiNET/MiNET/Blocks/Bedrock.cs [hunk 1]
```diff
@@ -1,3 +1,5 @@
+using MiNET.Utils;
```
Comment (PEMapModder) [inline src/MiNET/MiNET/Blocks/Bedrock.cs]: Why?

Comment (RandomAltThing) [inline src/MiNET/MiNET/Blocks/Bedrock.cs]: @PEMapModder MiNET.Utils contains the ItemStack method. Look for ItemStack.cs

Comment (RandomAltThing) [inline src/MiNET/MiNET/Blocks/Bedrock.cs]: @PEMapModder https://github.com/NiclasOlofsson/MiNET/blob/master/src/MiNET/MiNET/Utils/ItemStack.cs

**Pull request merged**